rdf_entity.module

```diff
@@ -29,6 +36,8 @@ function rdf_entity_entity_base_field_info_alter(&$fields, EntityTypeInterface $
     return;
   }
 
+  // @todo Now that graphs are entities, transform this field into an entity
```

Comment:
> I checked all the bundles and this properties is now empty. Isn't this replaced by the graph entities?
Not sure I understand your question. This is a base field added to all `rdf_entity` entities. The field is a URI (basically a string) and is stored in the 'value' columns. Now that the graphs are config entities we gould transform this field into a entity reference because now it stores the ID of an `rdf_entity_graph` entity. I've opened a follow-up in #15.